src/scripts/generate-cars.ts

```diff
@@ -0,0 +1,39 @@
+import { postCar } from './create-car';
+import { renderGarageFields } from './render-app';
+import getCurrentPageNumber from './utils';
+import { brandsAndModels, BrandAndModels } from './cars-and-models';
+
+async function createRandomCar(randomName: string, randomColor: string): Promise<void> {
+  const pageNumber = getCurrentPageNumber();
+  const data = {
+    name: randomName,
+    color: randomColor,
+  };
+  await postCar(data);
+  renderGarageFields(pageNumber);
+}
+
+function getRandomColor(): string {
+  return `#${Math.floor(Math.random() * 16777215).toString(16)}`;
+}
+
+function getRandomNumber(min: number, max: number): number {
+  const r = Math.random() * (max - min) + min;
+  return Math.floor(r);
+}
+
+function getRandomName(array: BrandAndModels[]): string {
+  const brandIndex = getRandomNumber(0, array.length);
+  const modelsArray = array[brandIndex].models;
+  const modelIndex = getRandomNumber(0, modelsArray.length);
+  const result = `${array[brandIndex].brand} ${array[brandIndex].models[modelIndex]}`;
+  return result;
```

Comment:
Local variable 'result' is redundant
```suggestion
  return `${array[brandIndex].brand} ${array[brandIndex].models[modelIndex]}`;
```